src/glossary.rst

```diff
@@ -967,3 +967,115 @@ Glossary
       See also:
 
       * :ref:`Graph Branching`
+
+source workflow
+   A source workflow is a :term:`cylc workflow` that has not yet been
+   :term:`installed` into the :term:`run directory`.
```

Comment:
Only thing is, the source workflow still exists separately after it has been installed.

Reply:
Yep, on second read this isn't very good, attempt two.